api/src/graphql/resolvers/index.js

```diff
@@ -106,45 +108,53 @@ const resolvers = {
     },
   },
   Mutation: {
-    createNeed: combineResolvers(
-      isAuthenticated,
-      (obj, { title }, { user, driver }) => createNeed(driver, { title }, user.email),
-    ),
+    createNeed: combineResolvers(isAuthenticated, (obj, { title }, { user, driver }) =>
+      createNeed(driver, { title }, user.email)),
     createResponsibility: combineResolvers(
       isAuthenticated,
       (obj, { title, needId }, { user, driver }) =>
         createResponsibility(driver, { title, needId }, user.email),
     ),
-    createViewer: combineResolvers(
-      isAuthenticated,
-      (obj, args, { user, driver }) => createViewer(driver, user.email),
-    ),
-    updateNeed: combineResolvers(
-      isAuthenticated,
-      (obj, args, { driver }) => updateReality(driver, args),
-    ),
-    updateResponsibility: combineResolvers(
-      isAuthenticated,
-      (obj, args, { driver }) => updateReality(driver, args),
-    ),
-    updateViewerName: combineResolvers(
-      isAuthenticated,
-      (obj, { name }, { user, driver }) => updateViewerName(driver, { name }, user.email),
-    ),
+    createViewer: combineResolvers(isAuthenticated, (obj, args, { user, driver }) =>
+      createViewer(driver, user.email)),
+    updateNeed: combineResolvers(isAuthenticated, async (obj, args, { driver, user }) => {
+      const realityData = await getRealityData(driver, args);
```

Comment:
Is it possible to give this const a name that helps better to explain how it's used? (Just a thought. Don't spend too much time on it 😄)

Reply:
Changed to emailData and getEmailData in the connector.